modules/ai/module.php

```diff
@@ -734,7 +734,6 @@ public function ajax_ai_get_image_to_image_mask( $data ) {
 			'prompt' => $data['payload']['prompt'],
 			'attachment_id' => $data['payload']['image']['id'],
 			'mask' => $data['payload']['mask'],
-			'image_base64' => $data['payload']['image_base64'],
```

Comment:
Where is the image itself?

Reply:
see the attachment id at L735 and its passed here - https://github.com/elementor/elementor/blob/main/modules/ai/connect/ai.php#L491